### PR TITLE
Add more missing translations

### PR DESCRIPTION
**Pull request by WiNloSt**
### Description

Add the missing translation reported in https://github.com/metabase/metabase/issues/33046.

Please note that from the original report
> 1. enterprise/frontend/src/metabase-enterprise/audit_app/lib/cards (in each file in each object the name field without t...)

Actually, not all `name` attributes should be translated. I only include the translations that are visible in our app.

### How to verify

Go to all places I've shown in the inline screenshots. For certain strings that are already translated, you'll see those strings being translated when changing language in the user settings. While there are some strings that need translation.

### Checklist

- [ ] ~Tests have been added/updated to cover changes in this PR~ No test since this is just adding missing translation

### CODE REVIEW

**Comment by WiNloSt** [comment]
This should be backported manually together with https://github.com/metabase/metabase/pull/33042, so we won't crash the  release branch.

File: frontend/src/metabase/admin/permissions/selectors/data-permissions/groups.ts [hunk 1]
```diff
@@ -1,25 +1,17 @@
...
-import { assoc } from "icepick";
 
 import Groups from "metabase/entities/groups";
 import { Group } from "metabase-types/api";
-import {
-  getGroupNameLocalized,
-  isAdminGroup,
-  isDefaultGroup,
-} from "metabase/lib/groups";
+import { isAdminGroup, isDefaultGroup } from "metabase/lib/groups";
```
Comment (WiNloSt) [inline frontend/src/metabase/admin/permissions/selectors/data-permissions/groups.ts:6]: Revert the code to where it is before https://github.com/metabase/metabase/pull/33035. The reason is explained in the comment in `frontend/src/metabase/admin/permissions/selectors/data-permissions/group-sidebar.ts`

File: frontend/src/metabase/collections/components/MoveCollectionModal/MoveCollectionModal.tsx [hunk 2]
```diff
@@ -23,7 +24,7 @@ const MoveCollectionModal = ({
 
   return (
     <CollectionMoveModal
-      title={`Move ${collection.name}?`}
+      title={t`Move "${collection.name}"?`}
```
Comment (WiNloSt) [inline frontend/src/metabase/collections/components/MoveCollectionModal/MoveCollectionModal.tsx:27]: Screenshot 2023-08-11 at 8 02 17 PM

Modify the string to contain double quotes as well so we reuse an existing translation from

https://github.com/metabase/metabase/blob/73cb16ee60c2d1742691fc0df5f7b05d9c24654a/frontend/src/metabase/collections/components/BulkActions.jsx#L97

File: frontend/src/metabase/entities/users/forms.js
Comment (WiNloSt) [inline frontend/src/metabase/entities/users/forms.js:1]: Screenshot 2023-08-11 at 9 24 47 PM

File: frontend/src/metabase/visualizations/components/settings/ChartSettingsTableFormatting.jsx [hunk 3]
```diff
@@ -324,7 +324,7 @@ const RuleEditor = ({
         value={rule.columns}
         onChange={e => onChange({ ...rule, columns: e.target.value })}
         isInitiallyOpen={rule.columns.length === 0}
-        placeholder="Choose a column"
+        placeholder={t`Choose a column`}
```
Comment (WiNloSt) [inline frontend/src/metabase/visualizations/components/settings/ChartSettingsTableFormatting.jsx:327]: Screenshot 2023-08-11 at 9 30 15 PM

File: frontend/src/metabase/visualizations/lib/settings/column.js
Comment (WiNloSt) [inline frontend/src/metabase/visualizations/lib/settings/column.js:1]: Screenshot 2023-08-11 at 9 33 26 PM

File: frontend/src/metabase/visualizations/visualizations/Gauge/Gauge.jsx [hunk 4]
```diff
@@ -115,7 +115,7 @@ export default class Gauge extends Component {
       readDependencies: ["gauge.segments"],
     },
     "gauge.segments": {
-      section: "Display",
+      section: t`Display`,
```
Comment (WiNloSt) [inline frontend/src/metabase/visualizations/visualizations/Gauge/Gauge.jsx:118]: Screenshot 2023-08-11 at 9 38 06 PM

File: enterprise/frontend/src/metabase-enterprise/audit_app/pages/AuditQueryDetail.jsx [hunk 5]
```diff
@@ -25,7 +25,7 @@ const AuditQueryDetail = ({ params: { queryHash } }) => (
 
       return (
         <AuditContent
-          title="Query"
+          title={t`Query`}
```
Comment (WiNloSt) [inline enterprise/frontend/src/metabase-enterprise/audit_app/pages/AuditQueryDetail.jsx:28]: There are multiple ways to come to this page, but one of the ways to get here is to:
1. Go to Admin > Audit > Databases > All Databases
2. Select any database
3. Click on any `ad-hoc` query

Screenshot 2023-08-15 at 5 03 03 PM

File: frontend/src/metabase/entities/users/forms.js [hunk 6]
```diff
@@ -9,22 +9,22 @@ export default {
...
-        placeholder: "Appleseed",
+        placeholder: t`Appleseed`,
         validate: validate.maxLength(100),
         normalize: lastName => lastName || null,
       },
       {
         name: "email",
         title: t`Email`,
-        placeholder: "nicetoseeyou@email.com",
+        placeholder: t`nicetoseeyou@email.com`,
```
Comment (kulyk) [inline frontend/src/metabase/entities/users/forms.js:27]: Do we really need to translate an email address?

**Review by kulyk — APPROVED** [review]
Great work, thanks! Left a comment about email address translation, please take a look

File: frontend/src/metabase/entities/users/forms.js (hunk 6 above)
Comment (ranquild) [reply frontend/src/metabase/entities/users/forms.js:27]: +1 likely not @WiNloSt

**Review by ranquild — APPROVED** [review]

File: frontend/src/metabase/entities/users/forms.js (hunk 6 above)
Comment (WiNloSt) [reply frontend/src/metabase/entities/users/forms.js]: Fixed in https://github.com/metabase/metabase/pull/33123/commits/9d82e2a679ffa0f62e6b48421115caff3c8ccbb4

**Pull request merged**